### DISCUSSION TITLE
ImportError for lemmy.pipe

### DISCUSSION THREAD

**Discussion by vrunm** [centre-for-humanities-computing/DaCy#199] [General]
I'm trying to use Lemmy as a SpaCy integration following the instructions [here.](https://spacy.io/universe/project/lemmy) However, when I try running import lemmy.pipe I get the error ImportError: cannot import name 'PRON_LEMMA' from 'spacy.symbols' .

I'm using python 3.8 and SpaCy 3.3.0 with the [DaCy pipeline](https://pythonawesome.com/dacy-the-state-of-the-art-danish-nlp-pipeline-using-spacy/).
I am using Spacy 3.3.0 and AllenNlp v1.5.0

I am running into this import issue:
`
cannot import name 'PRON_LEMMA' from 'spacy.symbols' (/opt/conda/lib/python3.7/site-packages/spacy/symbols.cpython-37m-x86_64-linux-gnu.so)`

How do I get around this?
Thank you for you input.

**Comment by KennethEnevoldsen** [comment]
Hi @vrunm I have answered in the issue. It is caused by Lemmy relying on spacy v2 and DaCy relying on v3. Sadly Lemmy is no longer updated so you can't use both packages together. However the next iteration of DaCy will improve upon the Lemmatizer

**Comment by vrunm** [comment]
Sure thanks.
